sky/clouds/aws.py

```diff
@@ -695,6 +695,54 @@ def _get_disk_specs(cls, disk_tier: Optional[str]) -> Dict[str, Any]:
             'custom_disk_perf': tier != 'low',
         }
 
+    @classmethod
+    def check_quota_not_zero(cls,
+                             region: str,
+                             instance_type: str,
+                             use_spot: bool = False) -> bool:
+        """
+        AWS-specific implmentation of check_quota_not_zero. The function works by
+        matching the instance_type to the corresponding AWS quota code, and then using
+        the boto3 Python API to query the region for the specific quota code.
+
+        Returns:
+            False if the quota is found to be zero, and True otherwise.
+        Raises:
+            ImportError: if the dependencies for AWS are not able to be installed.
+            botocore.exceptions.ClientError: error in Boto3 client request.
+        """
+
+        try:
+            # pylint: disable=import-outside-toplevel
+            import boto3
+            import botocore
+        except ImportError:
+            raise ImportError('Fail to import dependencies for AWS.'
+                              'Try pip install "skypilot[aws]"') from None
```

Comment:
This is not needed, as our upper-level caller will make sure the libraries exist before calling into the function.